modules/client/src/internal/constants.ts

```diff
@@ -33,3 +42,19 @@ export const UNAVAILABLE_DAO_METADATA: DaoMetadata = {
   description: "(the DAO metadata is not available)",
   links: [],
 };
+export const UNSUPPORTED_RELEASE_METADATA_LINK: PluginRepoReleaseMetadata = {
```

Comment:
These names are already long, but it's not clear that this applies to a plugin. Probably fine as it is, just pointing out (resolve)